django/contrib/redirects/migrations/0001_initial.py

```diff
@@ -27,10 +20,15 @@ class Migration(migrations.Migration):
                     help_text="This can be either an absolute path (as above) or a full URL starting with 'http://'.",
                     max_length=200, verbose_name='redirect to', blank=True
                 )),
+                # We need to add `domain` field to migrate sites references
+                # data. Some db backends add fields by creating new table
+                # and copying all fields which are in project state. This
+                # field helps preserve `sites` references without requiring
+                # dependency on 'django.contrib.sites' migrations.
+                ('site_id', models.IntegerField(null=True)),
```

Comment:
I am not so sure about that change. This is not an integer field, but a FK. It's not nullable, it has an index and some databases have other pseudo tables to reference a FK field.
I am not sure if this actually works using all Databases. I would recommend to be extra careful when changing old migrations.

Reply:
Though it may be valid concern, I really would like some example where it actually doesn't work.

The only reason this field is actually there is to preserve it in project state, it shouldn't affect existing database in any way.